README.md

```diff
@@ -256,6 +256,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Exspanse](https://exspanse.com) - MLOPS Platform to build, train and deploy ML models and AI solutions. The free plan allows the creation of unlimited projects, 5Gb of cloud storage, and five docker container images.
   * [ExtendsClass](https://extendsclass.com/rest-client-online.html) - Free web-based HTTP client to send HTTP requests.
   * [Export SDK](https://exportsdk.com) - PDF generator API with drag-and-drop template editor that provides an SDK and no-code integrations. The free plan has 250 monthly pages, unlimited users, and three templates.
+  * [Fern](https://buildwithfern.com) - Use your API definition to generate SDKs in popular languages, and generate API reference documentation webpages. Add Markdown pages to your API reference and host them with Fern for a full documentation solution. OpenAPI fully supported.
```

Comment:
This appears to be "Book a demo"ware rather than something useful you can sign up for?

Reply:
Here's the pricing page:
https://www.buildwithfern.com/pricing
They have separate free tiers for the docs product and for the sdk code generation product.

Reply:
Ok, so then only the doc part is free and SaaS like since that includes hosting right?

SDK generation doesnt appear to be SaaS based it just runs in your shell for the basic freatures?

Reply:
Yes, docs does include hosting, along with generating API reference pages from API definition files and generating other docs pages from Markdown and yml config.

SDK generation is also SaaS based as the shell command contacts their cloud-based service to generate the client library code. Free tier works for 20 endpoints, no registration needed.

Reply:
Just want to say, I really appreciate how thorough you are being with vetting the submission. Speaks for the quality of the list.

Reply:
Thank you appreciate your comments :)